src/test_all.py

```diff
@@ -140,7 +158,14 @@ def assert_language_name(name):
                     except KeyError:
                         raise Exception(f" * Looks like we have a repeated example here! "
                                         f"Merge outputs before removing duplicates. :-/ \n {instance}")
-
+                        
+                        
+            #Make sure classes are balanced
+            output=[ins['output'] for ins in instances]
+            outputs = sum(output, [])
+            skewness(outputs)
```

Comment:
Before running these skewness tests, you'd need to check if the number of unique outputs is not too many (say, it's less than 15).

Reply:
I've got a problem here. We had some classification tasks with more than 70 classes, for example task 481 here: https://github.com/swarooprm/CSE576HW/pull/111
Although checking some of the classes gives us a sense of skewness, but it might be inaccurate. On the other hand, checking all the different outputs in other categories is not necessary.

Reply:
Yeah, this is the best way. I'll change it now.